### PR TITLE
Fixed updating revision property on component change

### PR DESCRIPTION
**Pull request by AlexeyDavyd**

### CODE REVIEW

File: src/components/form/Form.js [hunk 1]
```diff
@@ -84,8 +84,8 @@ export default class FormComponent extends Component {
     }
 
     // Add revision version if set.
-    if (this.component.formRevision || this.component.formRevision === 0) {
-      this.formSrc += `/v/${this.component.formRevision}`;
+    if (this.component.revisions || this.component.revisions === 0) {
```
Comment (travist) [inline src/components/form/Form.js]: It should be singular. "revision"

**Comment by travist** [comment]
Why are we changing the property name? We have customers that have already built forms with this configured, so won't this break their forms if they upgrade? Could we make this reverse compatible so that it will work with either "formRevision" or "revision"?...

**Comment by travist** [comment]
@AlexeyDavyd I am still waiting for confirmation from you that this will not mess up forms that were built with 3.x builder and rendered in 4.x renderer with this patch. If you can confirm that there is no breaking change, then we will pull this in.

**Comment by AlexeyDavyd** [comment]
@travist After my checks I can confirm, that it shouldn't break existing forms.

**Pull request merged**